packages/api/cli/src/electron-forge.ts

```diff
@@ -39,13 +21,8 @@ program
   .command('package', 'Package the current Electron application')
   .command('make', 'Generate distributables for the current Electron application')
   .command('publish', 'Publish the current Electron application')
-  .on('command:*', async (commands) => {
-    if (!program._execs.has(commands[0])) {
-      console.error();
-      console.error(chalk.red(`Unknown command "${program.args.join(' ')}".`));
-      console.error('See --help for a list of available commands.');
-      process.exit(1);
-    } else if (!process.argv.includes('--help') && !process.argv.includes('--h')) {
+  .hook('preSubcommand', async () => {
+    if (!process.argv.includes('--help') && !process.argv.includes('--h')) {
```

Comment:
I noticed that the previous one is also like this. Is this expected?

```suggestion
    if (!process.argv.includes('--help') && !process.argv.includes('-h')) {
```